packages/ts-morph/src/compiler/types/Type.ts

```diff
@@ -233,9 +233,19 @@ export class Type<TType extends ts.Type = ts.Type> {
     }
 
     /**
-     * Gets the target type of a type reference if it exists.
+     * Gets the "target type".
```

Comment:
Sorry, this is having a bit of churn, but maybe we can put most of this information right in one or two sentences? Something like "Gets the generic type when the type is a type reference, returns itself when it's already the generic type, or otherwise returns undefined."?

I think we can get rid of the "Specifically" part if we just do that. I always find reading lists to understand functionality a bit harder to understand, but maybe it's just me.

Reply:
Yeah that's perfect!